stable/kubernetes-dashboard/values.yaml

```diff
@@ -80,8 +80,13 @@ ingress:
   ##
   # annotations:
   #   kubernetes.io/ingress.class: nginx
-  #   nginx.ingress.kubernetes.io/secure-backends: "true"
+
```

Comment:
Nitpicking, but why new line?

Reply:
No specific reason, I think I started to add a comment and then removed or something like that.